NEWS

```diff
@@ -29,6 +29,7 @@ PostGIS 3.0.0
   - #4215, Use floating point compare in ST_DumpAsPolygons (Darafei Praliaskouski)
   - #4155, Support for GEOMETRYCOLLECTION, POLYGON, TIN, TRIANGLE in
     ST_LocateBetween and ST_LocateBetweenElevations (Darafei Praliaskouski)
+  - #2767, Added "opt" and Boolean values for AddRasterConstraint (Sunveer Singh)  
```

Comment:
```suggestion
  - #2767, Documentation for AddRasterConstraint optional parameters (Sunveer Singh)
```

Reply:
note that you've added spaces at the end here, and tests may complain :)

Reply:
Thanks for suggestions